### PR TITLE
[plugin.video.espn-player] 1.0.1

### PR DESCRIPTION
**Pull request by emilsvennesson**
Watch content from ESPN Player's NCAA College Pass, Indycar Series and
ESPN Select.

### CODE REVIEW

File: plugin.video.espn-player/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
+    <import addon="script.module.requests" version="2.9.1"/>
+    <import addon="script.module.m3u8" version="0.2.10"/>
+    <import addon="script.module.xmltodict" version="0.9.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
```
Comment (phil65) [inline plugin.video.espn-player/addon.xml]: would be cool if you could add the new `<news>` element.

File: plugin.video.espn-player/default.py [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+    else:
+        for name, service in services.items():
+            parameters = {'action': 'main_menu', 'service': service}
+            add_item(name, parameters)
+        xbmcplugin.endOfDirectory(_handle)
+
+
+def main_menu(service):
+    listing = []
+    items = ['today', 'upcoming', 'archive', 'channels']
```
Comment (phil65) [inline plugin.video.espn-player/default.py]: please use language file.

File: plugin.video.espn-player/default.py [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+
+        add_item(item.title(), parameters)
+    xbmcplugin.endOfDirectory(_handle)
+
+
+def list_today(service):
+    now = datetime.now()
+    date_today = now.date()
+    abc = []
+    items = ['live', 'upcoming', 'archive']
```
Comment (phil65) [inline plugin.video.espn-player/default.py]: same here.

**Comment by emilsvennesson** [comment]
@phil65 Fixed!

**Pull request merged**